packages/paste-core/components/alert-dialog/src/AlertDialogFooter.tsx

```diff
@@ -47,7 +50,7 @@ export const AlertDialogFooter = React.forwardRef<HTMLDivElement, AlertDialogFoo
           <Button variant="secondary" onClick={onDismiss}>
             {onDismissLabel}
           </Button>
-          <Button variant={primaryVariant} onClick={onConfirm} disabled={destructive && onConfirmDisabled}>
+          <Button variant={primaryVariant} onClick={onConfirm} disabled={destructive && isConfirmDisabled}>
```

Comment:
suggestion (blocking): let's preserve the functionality of `onConfirmDisabled` here too

`disabled={destructive && (isConfirmDisabled || onConfirmDisabled)}`

Reply:
Roger that.